make/Makefile.build.mk

```
@@ -92,6 +92,16 @@ test-e2e:
 	@echo Running E2E tests
 	cd tests/e2e && source .kiali-e2e/bin/activate && pytest -s tests/
 
+## test-integration-setup: Setup go library for converting test result into junit xml
+test-integration-setup:
+	@echo Setting up Integration tests
+	go install github.com/jstemmer/go-junit-report@latest
+
+## test-integration: Run Integration test suite
+test-integration:
```

I think this should always do the setup. Since the setup is going to pull `@latest` we are going to want to always install this to make sure we have the latest (and thus make sure the integration tests still work with the latest)? So:
```suggestion
test-integration: test-integration-setup
```